libazureinit/src/http.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use reqwest::StatusCode;
+
+/// Set of StatusCodes that should be retried,
+/// e.g. 400, 404, 410, 429, 500, 503.
+pub const RETRY_CODES: &[StatusCode] = &[
```

Comment:
Similar to the `http` module, you can mark these constants as only available to the crate via `pub(crate)` so they're not automatically exposed if someone adds something we _do_ want public to the http module and then changes it from `pub(crate)` to `pub`